pkg/storage/image.go

```diff
@@ -285,11 +328,19 @@ func GetImageService(store storage.Store, defaultTransport string, insecureRegis
 		}
 	}
 
+	if len(additionalRegistries) == 0 {
+		// TODO(runcom): make docker.io a CONST
+		additionalRegistries = []string{"docker.io"}
```

Comment:
Should we add a default registry to replace docker.io and put this in the config file.  Then add additional registries, that way we don't hard code docker.io into the tool?  
I think we should use Trust/signing to block registries, so there is no reason to add that functionality here.